app/nexus.go

```diff
@@ -0,0 +1,545 @@
+package app
+
+import (
+	"context"
+	"fmt"
+	"github.com/temporalio/tcld/protogen/api/cloud/cloudservice/v1"
+	"github.com/temporalio/tcld/protogen/api/cloud/nexus/v1"
+	"github.com/temporalio/tcld/protogen/api/cloud/operation/v1"
+	"github.com/urfave/cli/v2"
+)
+
+type (
+	NexusClient struct {
```

Comment:
[Nit] any reason this type needs to be exported?

Reply:
The fx file that exports the client is in another package (`cmd/tcld`), therefore it needs to be exported.